exercises/dnd-character/lib/dnd_character.ex

```diff
@@ -0,0 +1,15 @@
+defmodule DndCharacter do
+  @spec modifier(pos_integer) :: integer
+  def modifier(score) do
+    # code here
+  end
+
+  @spec ability() :: pos_integer
+  def ability() do
```

Comment:
I don't think this codebase uses empty parentheses for function definitions with zero arguments. I'd like to stay consistent.